package.list.ros2

```diff
@@ -6,3 +6,4 @@ ros_base
 ros_core
 ros_workspace
 vision_msgs
+navigation2
```

Comment:
Please sort in alphabetical order

Reply:
Done in https://github.com/seqsense/aports-ros-updater/pull/83/commits/04f40d6873ea8edc404121d0d6325018e2f6a9f3